doc/requirements/sequential_cutoff

```diff
@@ -0,0 +1,234 @@
+id: threshold_all_cores
+---
+
+The management tool shall allow to specify sequential cutoff threshold
+for all cores associated with given cache instance at once.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed sequential cutoff theshold values
+id: threshold_values
+---
+
+It shall be possible to specify sequential cutoff threshold value in range
+from 1 to 4194181 KiB inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default sequential cutoff threshold value
+id: threshold_default
+---
+
+The default sequential cutoff threshold shall be 1024 KiB.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Setting sequential cutoff promotion count for core
+id: promotion_count_core
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for given core.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: >-
+  Setting sequential cutoff promotion count for all cores in given
+  cache instance
+id: promotion_count_all_cores
+---
+
+The management tool shall allow to specify sequential cutoff promotion count
+for all cores associated with given cache instance at once.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Allowed sequential cutoff promotion count values
+id: promotion_count_values
+---
+
+It shall be possible to specify sequential cutoff promotion count value
+in range from 0 to 65535 inclusive.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Default sequential cutoff promotion count value
+id: promotion_count_default
+---
+
+The default sequential cutoff promotion count shall be 8.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Loading sequential cutoff configuration on cache load
+id: loading_configuration
+---
+
+All sequential cutoff parameters shall be loaded from cache metadata for each
+core on cache load.
+
+--------------------------------------------------------------------------------
+--------------------------------------------------------------------------------
+title: Sequential cutoff stream detection
+id: operation
+---
+
+The sequential cutoff shall detect streams of sequential IO requests.
```

Comment:
"While the sequential cutoff policy is set to full or always, sequential cutoff shall detect streams of sequential IO requests."

Reply:
This is specified in another requirement.

Reply:
Sorry, I forgot to remove this comment while reviewing. Of course you're right.